### PR TITLE
Small improvements in readme

### PR DESCRIPTION
**Pull request by 304NotModified**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ website for the NuGet client. For information about the NuGet clients, visit htt
     ```
     Update-Database -StartUpProjectName NuGetGallery
     ```
-If this fails, you are likely to get more useful output by passing -Debug than -Verbose.
+If this fails, you are likely to get more useful output by passing `-Debug` than `-Verbose`.
 
-6. Change the value of `Gallery.ConfirmEmailAddresses` to false in `src\NuGetGallery\Web.Config`, this is required to upload the packages after registration.
+6. Change the value of `Gallery.ConfirmEmailAddresses` to `false` in `src\NuGetGallery\Web.Config`, this is required to upload the packages after registration.
```
Comment (maartenba) [inline README.md]: Maybe add another note here: by default e-mail are written to file system. Whenever the gallery sends out an e-mail (e.g. for user confirmation) the email ends up in the ~/App_Data folder.

You can set the Gallery.SmtpUri option in web.config too, to use a real SMTP server, for example:
 smtp://user:password@smtpservername:25

Comment (304NotModified) [inline README.md]: Good note! I will add it!

**Comment by 304NotModified** [comment]
@maartenba is there a way to enable the statistics locally? As I have seen from the code, there are some `.json` files (containing the stats) needed for that?

**Comment by maartenba** [comment]
Correct. There is no good way to do this currently. The Operations projects do contain some scripts that can read from the PackageStatistics table and generate these files, however the gallery no longer writes anything to that table when a download happens.

**Comment by 304NotModified** [comment]
Can't we have some (old) example statistics / json files? Because a cannot test any changes made in the stats pages.

**Comment by maartenba** [comment]
Here are a few example files: https://www.dropbox.com/s/wol2xjk2mcz56g4/stats.zip?dl=0

**Comment by 304NotModified** [comment]
@maartenba thanks! Got it working. But I had to write my own `IReportService` ? 

Anyway, maybe a bit off topic but,

- why are those files not included in the GitHub repos?
- I'm a bit surprised that this information isn't in a database? IMO a database call can be cheaper then reading a (json) file. 
- Ever more surprised that this information isn't cached (in memory)? Or do I miss some config?

**Comment by maartenba** [comment]
@304NotModified Let's discuss that in a separate thread. Reason is we have a slightly bigger setup that sources stats from blobs (that come from our CDN). Ideally we should log package requests into the `PackageStatistics` table (based on a config setting so we can turn it off for nuget.org) and create an IReportService that generates stats based on that table for local gallery installs.

File: README.md (hunk 1 above)
Comment (maartenba) [inline README.md]: Once added let me know and I'll merge this PR.

**Comment by 304NotModified** [comment]
OK, I will move the discussion in the future. 

I will also push a new commit once the web hook has been fixed (#2715)

**Comment by maartenba** [comment]
Just push, no need to wait for the web hook (has to be fixed but shouldnt be a requirement)

**Pull request merged**